addon/src/lib/reducers/sideEffects.js

```diff
@@ -181,6 +183,20 @@ export function reducer(
         installManager.syncInstalled();
       };
 
+    case actions.BROWSER_STARTUP.type:
```

Comment:
Yeah, sorry, initialization isn't consistent as you can see in my indecision with main vs the SELF_* actions and side effects. Ideally, I'd like to see it all in one place, all actions with just a tiny kick in main or all main no actions.

Lets keep it like this for now, and bikeshed async :)